<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>657</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>doIhave Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Search">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>270</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 162, 232);

color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 162, 232);</string>
    </property>
    <property name="maxLength">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/Imagens/doIhave_Logo.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_catwindow">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 162, 232);</string>
    </property>
    <property name="text">
     <string>View List</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Roboto&quot;;
color: rgb(0, 162, 232);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>320</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 162, 232);</string>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 162, 232);
font: 9pt &quot;Roboto&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
border-color: rgb(0, 162, 232);
font: 10pt &quot;Roboto&quot;;
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_List"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCredits"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCategories"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_List"/>
   <addaction name="actionSave"/>
   <addaction name="actionCategories"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/Imagens/document_save.png</normaloff>:/new/prefix1/Imagens/document_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
  </action>
  <action name="actionAdd_List">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/Imagens/add.png</normaloff>:/new/prefix1/Imagens/add.png</iconset>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
  </action>
  <action name="actionCredits">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/Imagens/help_about.png</normaloff>:/new/prefix1/Imagens/help_about.png</iconset>
   </property>
   <property name="text">
    <string>Credits</string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
  </action>
  <action name="actionCategories">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/Imagens/document_preview_archive.png</normaloff>:/new/prefix1/Imagens/document_preview_archive.png</iconset>
   </property>
   <property name="text">
    <string>View List</string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
